_blog/why-null-models-crispr.md

```diff
@@ -0,0 +1,84 @@
+
+Experiments can chip away at the first question. In the original study, they found in addition to this broad distribution that certain regions of the phage genome were more likely to be a source for spacers. Based on this, they concluded that the high-abundance spacers were special: their sequence makes them more likely to be high abundance. That's one piece of evidence that spacer sequences matter, and it's supported by other experiments. Here’s a figure from another experiment that looked specifically at the process of spacer acquisition - they infected a population of bacteria with phages, then sequenced new spacers 24 hours later without leaving the bacteria lots of time to grow and die. They performed the same experiment twice (two 'replicates'), and this plot shows the frequency of spacer sequences in the first replicate versus the frequency of the same spacer sequences in the second replicate. The data points are strongly correlated: they fall on or near a straight diagonal line with slope 1. This means that if a spacer sequence was highly abundant in the first experiment, it is also generally highly abundant in the second experiment, suggesting that there is indeed something about the spacer itself that destines it for high or low abundance[^3]. 
+
+[^3]: This [theory paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005486) included variable spacer effectiveness as an ingredient to get broad abundance distributions.
+
+<p align="center">
+<img src="/img/posts/why-null-models-crispr/Heler2019.svg" alt="Heler2019" width="450px" style="padding:5px;">
+</p>
+
+So we're done, right? We know why some spacers are highly abundant and others are not, and if we see a broad abundance distribution we know the cause. Or do we? We've answered the question "does process A cause result B", but what about the other direction? Remember where we started: to figure out what's going on when we observe something, we have to rule out other explanations. Could there be other explanations for a spacer abundance distribution that looks like this that would make it hard to say for sure that B is evidence of A?
```

Comment:
I guess you haven't used it in these words, but in layman's terms people might recognize this as correlation vs causation. Don't know if this needs to be included anywhere, I think you're clear enough.

Reply:
Oh yeah, that's a great point! I'll leave that out for now, but you've got me thinking!